setup.cfg

```diff
@@ -16,15 +16,14 @@ license = MIT
 classifiers =
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
 long_description_content_type = text/x-rst
 
 [options]
-python_requires = >=3.5.4
+python_requires = >=3.6
```

Comment:
Just to be extra sure and match what we use for CI :)

```suggestion
# When updating this value, make sure our CI matrix include the minimum version.
python_requires = >=3.6.7
```

Reply:
```suggestion
# When updating this value, make sure our CI matrix includes a matching minimum version.
python_requires = >=3.6.7
```